source/cli.rst

```diff
@@ -86,6 +138,16 @@ Description
 
 Debug GLPI cache.
 
+Arguments/Options
+****************
+
+There are no arguments for this command
+
+Options:
+
+- ``-k``, ``--key`` (required) - Cache key to debug. (no default)
+- ``-c``, ``--context`` (required) - Cache context to clear (i.e. 'core' or 'plugin:plugin_name'). (default = core)
```

Comment:
`InputOption::VALUE_REQUIRED` means that a value is required when the option is used, but it does not mean that the option itself has to be used.